firmware-binaries/demos/switch-demo1-mu/src/main.rs

```diff
@@ -27,32 +39,40 @@ fn main() -> ! {
         uwriteln!(uart, "{:?}", transceivers.statistics(channel)).unwrap();
     }
 
-    uwriteln!(uart, "Centering buffer occupancies").unwrap();
-    for (i, eb) in [
-        &INSTANCES.elastic_buffer_0,
-        &INSTANCES.elastic_buffer_1,
-        &INSTANCES.elastic_buffer_2,
-        &INSTANCES.elastic_buffer_3,
-        &INSTANCES.elastic_buffer_4,
-        &INSTANCES.elastic_buffer_5,
-        &INSTANCES.elastic_buffer_6,
-    ]
-    .iter()
-    .enumerate()
-    {
+    // Keep centering elastic buffers to avoid over/under-flows. Keep track of
+    // number of frames changed while centering.
```

Comment:
```suggestion
    // Keep centering elastic buffers to avoid over/under-flows. Keep track of
    // number of frames changed while centering.
    //
    // TODO: Implement asynchronous communication. The idea is that we center
    //       the elastic buffers and capture the UGNs. From that point on, the
    //       links can be used for arbitrary communication. We'll keep on
    //       moving the buffers to their midpoints to prevent overflows (given
    //       that clock control still might be correcting clocks), so we'll
    //       use `eb_changes` to calculate the final UGN once we determine
    //       the whole network is up and therefore stop modifying the buffer
    //       occupancy.
```